### PR TITLE
Added button for exporting PDFs

### PR DESCRIPTION
**Pull request by mikeyh-aexp**

### CODE REVIEW

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx [hunk 1]
```diff
@@ -1,8 +1,18 @@
 import * as React from 'react';
 import {Panel} from 'react-bootstrap';
 import moment from 'moment-timezone';
+// import { render } from "react-dom";
+// import { renderToString } from "react-dom/server";
```
Comment (henryleduc) [inline AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx]: Do we need these imports? can we remove the comments?

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx [hunk 2]
```diff
@@ -48,8 +58,18 @@ export default class SearchResult extends React.Component {
     getResultsHeader(results) {
         return results.length > 0
             ? (results.length === 1
-                ? <h3>{`${results.length} result`}</h3>
-                : <h3>{`${results.length} results`}</h3>)
+                ? <div className="result-heading"><h3>{`${results.length} result`}</h3><button onClick={this.jsPdfGenerator}>Download PDF</button></div>
+                : <div className="result-heading"><h3>{`${results.length} result`}</h3><button onClick={this.jsPdfGenerator}>Download PDF</button></div>)
```
Comment (henryleduc) [inline AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx]: Previously the string was 'results' is there a reason for this change?

File: AgentDiscoveries-Frontend/app/src/components/todays-code-page.jsx [hunk 3]
```diff
@@ -32,14 +32,12 @@ export default class TodaysCodePage extends React.Component {
         var moscow = london.clone().tz('Europe/Moscow');
```
Comment (henryleduc) [inline AgentDiscoveries-Frontend/app/src/components/todays-code-page.jsx]: Are these edits to the home page relevant/ where they needed?

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx (hunk 2 above)
Comment (mikeyh-aexp) [reply AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx]: Yes, this change adds the 'Download PDF' button to the search results beside the h3 heading.

File: AgentDiscoveries-Frontend/app/src/components/todays-code-page.jsx (hunk 3 above)
Comment (mikeyh-aexp) [reply AgentDiscoveries-Frontend/app/src/components/todays-code-page.jsx]: Yes, these were needed to pass npm linting

File: AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx (hunk 1 above)
Comment (mikeyh-aexp) [reply AgentDiscoveries-Frontend/app/src/components/search-forms/search-result.jsx]: These have now been deleted and committed

**Pull request merged**